custom/lib_portal/models/res_partner.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = 'res.partner'
+    
+    def _isMember(self):
+        """
+        this Function uses to get the ratification amount in words.
+        """
+        return True
+
+    dob = fields.Date(string='Date of Birth')
+    #dod = fields.Date('Date of Death') 
+    languages_known = fields.Many2many(string='Known Languages')
```

Comment:
* you need to specify the model this many2many field is linking here

Reply:
I have changed it to languages_known = fields.One2many('res.lang',string='Known Languages') .I have a doubt here.So the relation could be one to many right instead of Many2many as one partner instance can have many languages?